core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -550,9 +550,16 @@ oppia.factory('explorationRightsService', [
         var that = this;
 
         requestParams.version = explorationData.data.version;
-        var explorationRightsUrl = (
-          '/createhandler/rights/' + explorationData.explorationId);
-        $http.put(explorationRightsUrl, requestParams).then(function(response) {
+        var requestUrl = '';
```

Comment:
saveChangeToBackend seems to be a blanket function to make any change with respect to exploration rights in this _explorationRightsService_. Is introducing other public functions a good idea?

By "_each backend URL_", you mean _/createhandler/rights/_ and _/createhandler/publish/_ ?

By breaking up requestParams, you mean breaking in frontend, right? Because ExplorationRightsHandler expects 5 values from requestParams.